src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml

```diff
@@ -0,0 +1,175 @@
+                  12 (Admin        2023-03-09 12:23:11     2)   <title></title>
+                  63 (Admin        2023-03-13 16:16:05     3)   <description>Description 2 added...</description>
+                  44 (Admin        2023-03-09 17:39:39     4)   <devel project="home:Admin" package="package_2"/>
+                  11 (Admin        2023-03-09 12:14:58     5) </package>
+    '400':
+      description: Bad Request.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Bad Revision:
+              description: Passing `rev=invalid_revision`.
+              value:
+                code: 400
+                summary: bad revision 'invalid_revision'
+            Not Boolean:
+              description: Passing `meta=not_boolean`.
+              value:
+                code: 400
+                summary: not boolean
+            Unknown View:
+              description: Passing `view=invalid_view`.
+              value:
+                code: 400
+                summary: unknown view 'invalid_view'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      description: Not Found.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Unknown Project:
+              value:
+                code: unknown_project
+                summary: "Project not found: home:some_project"
+            Unknown Package:
+              value:
+                code: unknown_package
+                summary: "Package not found: home:some_project/some_package"
+            Unknown Revision:
+              value:
+                code: 404
+                summary: revision '66' does not exist
+                details: 404 revision '66' does not exist
+  tags:
+    - Sources - Packages
+
+put:
+  summary: Write package meta file.
+  description: Write package's meta file. Create the package if it doesn't exist.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: comment
+      schema:
+        type: string
+      description: Comment that explains the changes you made in the meta file.
+      example: Fix the title of the package.
+  requestBody:
+    description: Package meta definition.
+    content:
+      application/xml; charset=utf-8:
+        schema:
+          $ref: '../components/schemas/source/package_meta.yaml'
+        example:
+          name: package_1
+          title: ''
+          description: Test package
+          devel:
+            project: home:Admin
+            package: package_2
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '400':
```

Comment:
No matter the 400 example I choose, Swagger UI is displaying

```
<?xml version="1.0" encoding="UTF-8"?>
<notagname/>
```

Reply:
Also fixed. Thanks!